packages/cloud_firestore/android/src/main/java/io/flutter/plugins/firebase/cloudfirestore/CloudFirestorePlugin.java

```diff
@@ -307,4 +409,73 @@ public void onFailure(@NonNull Exception e) {
         }
     }
   }
+
+  private static class TransactionTask extends AsyncTask<Void, Void, Void> {
+
+    private String operation;
```

Comment:
Rather than using string constants for transaction operations, I think inline callbacks would be more consistent with what we're doing with other plugins.